src/components/home/home.tsx

```diff
@@ -1,42 +1,118 @@
                        <div className = 'home__logos-container'>
                           <img className = 'home__device-logo' src = {playstationLogo} alt = "Playstation Logo"/>
                           <p className = 'home__text'>Playstation 5</p>
                        </div>
-                       <div className = 'home__logos-container'>
-                          <img className = 'home__device-logo' src = {xboxLogo} alt = "Xbox logo"/>
-                          <p className = 'home__text'>XBox One</p>
-                       </div>  
+                    </Link>
+                       <Link className = 'home__link-container' to = {ROUTES.PRODUCTS}>
+                            <div className = 'home__logos-container'>                          
+                                <img className = 'home__device-logo' src = {xboxLogo} alt = "Xbox logo"/>
+                                <p className = 'home__text'>XBox One</p>
+                            </div> 
+                       </Link>   
                    </div>
                </div>
                <div>
                    <p className = 'home__title'>New games</p>
-                   <div>
-                       <img src="" alt="" />
-                       <img src="" alt="" />
-                       <img src="" alt="" />
-                   </div>
+                   <div className = 'home__game-cards'>
+                        <GameCard 
+                            backgroundImage = {overwatch} 
+                            description = 'Overwatch is a colorful team-based action game starring a diverse cast of powerful heroes. Travel the world, build a team, and contest objectives in exhilarating 6v6 combat.'
+                            ageLimit = '12 +'
+                        />
+                        <GameCard 
+                            backgroundImage = {minecraft} 
+                            description = 'Minecraft is a sandbox game. There is a virtual land where users can create their own worlds and experiences, using building blocks, resources discovered on the site and their own creativity.'
+                            ageLimit = '3 +'
+                        />
+                        <GameCard 
+                            backgroundImage = {terraria} 
+                            description = 'Terraria is a 2D sandbox game with gameplay that revolves around exploration, building, crafting, combat, survival, and mining, playable in both single-player and multiplayer modes.'
+                            ageLimit = '6 +'
+                        />
+                    </div>
+                    <div>
+                        <p className = 'home__title'>Top rated games</p>
+
+                        <div className = 'home__game-cards'>
+                            {topGames.map((item: IGamesArrray) => {
+                            return (
+                                <div key = {item.name}>
```

Comment:
in some cases item.name can be the same. It will be enough to add the second attribute index to function, and each time put index to key